Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM alpine
+FROM alpine 
 LABEL maintainers="Kubernetes Authors"
 LABEL description="CSI External Attacher"
 
-COPY csi-attacher csi-attacher
+COPY ./deploy/docker/csi-attacher csi-attacher
```

Comment:
in Makefile you put the binary into `./bin/csi-attacher`. Who copies it to `/deploy/docker/csi-attacher`?

Reply:
In other words, Dockerfile should probably use directly `bin/csi-attacher`, without copying it anywhere else.

Reply:
done